src/pages/[platform]/build-a-backend/data/working-with-files/index.mdx

```diff
@@ -275,12 +367,39 @@ const updatedSong = await client.models.Song.update({
   id: song.id,
   coverArtPath: null,
 });
-
 ```
+</InlineFilter>
+
 ### Remove the record association and delete the file
 
 The following example removes the file from the record, then deletes the file from S3:
 
+<InlineFilter filters={["swift"]}>
+```swift title="ContentView"
```

Comment:
Can you add a space between this code block and the opening InlineFilter tag? It looks like its causing the content to not show up:

<img width="784" alt="Screenshot 2024-06-17 at 11 37 07 AM" src="https://github.com/aws-amplify/docs/assets/376920/49ce7ad2-f3e4-48b7-90d0-68ba72bcbe35">